docs/ipac/nexsci/nasa_exoplanet_archive.rst

```diff
@@ -86,6 +86,27 @@ For example, a full table can be queried as follows:
     10155286  K07988.01              ...     LS+MCMC     0.092 296.76288,47.145142
     10156110  K07989.01              ...     LS+MCMC       0.0 297.00977,47.121021
 
+
+A list of accessible tables can be found in the `TAP_TABLES` attribute:
```

Comment:
so sphinx is not failing on it
```suggestion
A list of accessible tables can be found in the ``TAP_TABLES`` attribute:
```

Reply:
I still don't quite understand the workflow, I guess, since I keep making the same mistake :D
How do I send a PR to my own fork? Is that what you mean?

My workflow now is:
1. Ensure myfork is up-to-date with astroquery/main by syncing in Github
2. Pull myfork/main locally
3. Create a local fixbranch for code fix/update
4. Make fix/update
5. Push fixbranch to myfork
6. Submit PR

Reply:
instead of 1-2, you could fetch astroquery/main. Then in step 3, create a local fixbranch from that one, rather than from anything on your fork.

Reply:
Ah, I see. I'll try that next time. Thanks!